corehq/apps/registration/urls.py

```diff
@@ -21,6 +22,8 @@
     url(r'^domain/$', RegisterDomainView.as_view(), name='registration_domain'),
     url(r'^domain/confirm(?:/(?P<guid>\w+))?/$', confirm_domain,
         name='registration_confirm_domain'),
+    url(r'^mobile_reminder/', send_desktop_reminder,
```

Comment:
Nitpicky, but it'd be slightly more readable if this were either always a "mobile" reminder or always a "desktop" reminder.